protocols/identify/src/protocol.rs

```diff
@@ -105,7 +152,7 @@ impl<C> OutboundUpgrade<C> for Identify
 where
     C: AsyncRead + AsyncWrite + Unpin + Send + 'static,
 {
-    type Output = Info;
+    type Output = PartialInfo;
```

Comment:
This should not be needed, for the regular identify protocol, we always send full `Info`s.